### ISSUE TITLE
Adding the cop that was offended in the error message will help searchability

### ISSUE THREAD

**Issue by vmeyet** [bbatsov/rubocop#4473]
The error message given by the cop should state which cop was offended. Right now it only state how to fix the issue.

Having this will help searchability on search engines or this repo (to see what are the different options, why this cop was chosen to be added etc). This will save developer time

##  Example of use case

As a developer, when I update the rubocop gem and run the executable against the my ruby files. A lot of new offenses are found. If I want to deactivate a particular cop, I have to google the error message and crawl through this repo to find the name of the cop in order to deactivate it.

## Expected behavior

```
lib/core_ext/rails.rb:174:68: C [Style/FormatStringToken]: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
```

## Actual behavior

```
lib/core_ext/rails.rb:174:68: C: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
```

## RuboCop version

```
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.1 x86_64-darwin16)
```

I hope this issue make sense. :)

**Comment by benoitjoseph** [comment]
👍

**Comment by backus** [comment]
Try the `--display-cop-names` flag

**Comment by vmeyet** [comment]
didn't know about this flag (:

Thanks. I think this can be closed then
